Rakefile

```diff
@@ -1,5 +1,11 @@
 # frozen_string_literal: true
 
+begin
+  require "parallel_tests/tasks"
+rescue LoadError
+  puts "No parallel_tests available. Perhaps running in production mode?"
```

Comment:
I'd rather we just output nothing if this happens, like we do in the rspec block below.

It shouldn't impact anything but we are reliant on rake being run in production mode in a number of different scenarios, such as when we deploy, or when we run automated jobs in production.

If we are running parallel tests and the require fails, we'll know about it anyway when the next step fails. So as rake is more often used in a production mode, I'd prefer not have this message show each time we do.

Reply:
```suggestion
  # No parallel_tests available. Perhaps running in production mode. 
```